docs/gt4py/quickstart.rst

```diff
@@ -128,10 +128,10 @@ function with the ``stencil`` decorator provided by GT4Py.
 The ``stencil`` decorator generates code in Python or C++ depending on the ``backend`` specified by name.
 Currently, the following backends are available:
 
-* ``"gtc:numpy"``: a vectorized python backend
-* ``"gtc:gt:cpu_kfirst"``: a backend based on GridTools code performance-optimized for x86 architecture
-* ``"gtc:gt:cpu_ifirst"``: a GridTools backend targeting many core architectures
-* ``"gtc:gt:gpu"``: a GridTools backend targeting GPUs
+* ``"numpy"``: a vectorized python backend
+* ``"gt:cpu_kfirst"``: a backend based on GridTools code performance-optimized for x86 architecture
+* ``"gt:cpu_ifirst"``: a GridTools backend targeting many core architectures
+* ``"gt:gpu"``: a GridTools backend targeting GPUs
 * ``"gtc:gpu"``: a backend targeting NVIDIA GPUs using CUDA and some GridTools
```

Comment:
This should have said `"gtc:cuda"` before, now `"cuda"`?

Reply:
Good catch! Changed.